Ruins/src/main/java/atomicstryker/ruins/common/RuinTemplate.java

```diff
@@ -906,7 +907,7 @@ else if (line.startsWith("biomeTypesToSpawnIn"))
                 else if (line.startsWith("weight"))
                 {
                     String[] check = line.split("=");
-                    weight = Integer.parseInt(check[1]);
+                    weight = Math.max(Double.parseDouble(check[1]), 0);

```

Comment:
Does a weight of 0 do anything?

Reply:
It means the template won't be selected for worldgen spawing. Whether there's a point to that or not, I can't say (for temporarily disabling a template, maybe?), but it's always been a valid value and still is. The only thing I'm doing now is guarding against negative weights, which used to get through and screw up the selection process.

I can't think of any particularly compelling reason to disallow 0--it works, and has the expected effect.